src/content/general/architecture/capi/index.md

```diff
@@ -0,0 +1,227 @@
+
+To learn more about organizations please read [the related documentation](https://docs.giantswarm.io/general/organizations/).
+
+*RBAC operator*
+
+We have built an RBAC operator with the goal of maintaining up to date permissions between the different organizations and users on the management cluster so [you can isolate your teams and ensure access level in a granular way](https://docs.giantswarm.io/ui-api/management-api/authorization/).
+
+*DNS operator*
+
+We built an operator to manage the DNS entries for all the endpoints we need to expose to our customers. All our management clusters have a base domain used to allocate the API, OIDC endpoint or UI. This operator listens to cluster resource and provides the right records.
+
+*Deletion Blocker Operator*
+
+Kubernetes ensures resources are deleted in the right order by using `finalizers`, a way to create dependencies between resources. In specific cases, some CRs don't leverage finalizers, which can stop the deletion process when they are deleted too early and leave the cluster in an inconsistent state. Deletion Blocker Operator ensure that specific CRs aren't deleted until specific conditions are met.
+
+*App operator*
+
+The [App Platform](https://docs.giantswarm.io/app-platform/) is a system we built to deliver Cloud Native Apps in a multi cluster fashion. App operator is the main code running in management clusters to manage App custom resources and make sure the applications and configuration are delivered correctly across the different workload clusters.
+
+*Cluster API operators*
+
+In [Cluster API](https://cluster-api.sigs.k8s.io/) there is a set of generic operators that all Cluster API providers run to address the common bootstrap and management actions on a Kubernetes cluster lifecycle which include:
+
+- `Kubeadm Bootstrap`controller which is in charge of providing a cloud init configuration to turn a machine into a Kubernetes node.
+- `Kubeadm Control Plane` controller that manages the lifecycle of the control plane nodes and provides access to the API.
+- `CAPI manager` controller that manages cluster and machine resources.
+
+*Cluster Provider operator*
+
+In Cluster API there is a set of common providers defined above, but to manage the real infrastructure we need to run a specific controller that knows about the cloud provider's API. It acts as a bridge and reconciles the provider specific configuration to create the necessary infrastructure for a Kubernetes cluster (i.e. VMs, Load Balancers...). This controller provisions, updates and deletes all resources through API. Every provider has its own lingo and methods to create a machine or provision a network. The operator reconciles the `Infrastructure` custom resources of the specific provider parsing the desired state into API requests and ensuring the provider resources are always in sync with the resource definitions stored in the management cluster.
```

Comment:
```suggestion
In Cluster API there is a set of common providers defined above, but to manage the real infrastructure we need to run a specific controller that knows about the cloud provider's API. It acts as a bridge and reconciles the provider specific configuration to create the necessary infrastructure for a Kubernetes cluster (i.e. VMs, Load Balancers...). This controller provisions, updates and deletes all resources through API. Every provider has its own lingo and methods to create a machine or provision a network. The operator reconciles the infrastructure resources of the specific provider parsing the desired state into API requests and ensuring the provider resources are always in sync with the resource definitions stored in the management cluster.
```

Since there is no resource named "Infrastructure".